src/_base/harness/scripts/rebuild.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+run docker-compose down --rmi local --volumes --remove-orphans --timeout 120
+
+passthru ws cleanup built-images
+
+run rm -f .my127ws/.flag-built
```

Comment:
just a thought, I probably would actually want to delete .flag-console-built as well, as that should be rebuilt, just not the mutagen volume